lib/resque/plugins/retry/hooks.rb

```diff
@@ -0,0 +1,62 @@
+module Resque
+  module Plugins
+    module Retry
+      module Hooks
+        # Returns a sorted list of hook methods, according to the hook prefix.
+        #
+        # @param hook_prefix [String]
+        #
+        # @returns [Array<Symbol>]
+        #
+        # @api private
+        def get_hook_names(hook_prefix)
```

Comment:
Do these need to be sorted? While I believe I understand the philosophy behind it I had to ask.